scripts/superdesk-authoring/services.js

```diff
@@ -1,7 +1,55 @@
 (function() {
 'use strict';
 
-angular.module('superdesk.authoring').service('renditions', RenditionsService);
+angular.module('superdesk.authoring')
+.service('renditions', RenditionsService)
```

Comment:
I think these two lines should be indented 4 spaces to the right based on most of our codebase (as far as I've seen).